### PR TITLE
Flake8 phase8

### PR DESCRIPTION
**Pull request by jochman**
fixes: https://app.circleci.com/pipelines/github/demisto/content/14986/workflows/31aa6a93-61b0-45de-b522-7406e78a6fb2/jobs/56786/parallel-runs/1?filterBy=FAILED

connected: https://github.com/demisto/content/pull/7338
- [x] Ready
CiscoASA
GooglePubSub
Looker
MicrosoftGraphMail

### CODE REVIEW

**Comment by Itay4** [comment]
check why cisco asa fails

**Comment by jochman** [comment]
Failed on Phising (mailbox is full). We can merge @Itay4

File: Packs/Slack/Integrations/Slack/Slack.yml [hunk 1]
```diff
@@ -521,7 +521,7 @@ script:
     - contextPath: Slack.User.Email
       description: The email address of the user.
       type: String
-  dockerimage: demisto/slack:1.0.0.7327
+  dockerimage: demisto/slack:1.0.0.8976
```
Comment (Itay4) [inline Packs/Slack/Integrations/Slack/Slack.yml:524]: revert this change

**Comment by Itay4** [comment]
@ShahafBenYakir gmail and cisco asa tests fail here, see if you want us to skip them here

**Comment by ShahafBenYakir** [comment]
I am not recommending skipping, better to force merge or investigate.

**Comment by jochman** [comment]
> I am not recommending skipping, better to force merge or investigate.

Gmail is failing on quota.

**Comment by jochman** [comment]
Cisco ASA is failing, the issue here: https://github.com/demisto/etc/issues/25741
It passes on nightly because we're using mocks there, it'll fail when it will try to record new one (like after this pr.)
adding it to skipped.

**Comment by Itay4** [comment]
build fails on non relavent test
merging
